bors.toml

```
@@ -19,6 +18,8 @@ status = [
   "tools-build-success",
   ]
 
+block_labels = "Process: missing approvals"
```

this maybe needs to be a list.

```suggestion
block_labels = [ "Process: missing approvals" ]
```